### ISSUE TITLE
OpenXR support

### ISSUE THREAD

**Issue by ASchneiderBR** [baffler/OBS-OpenVR-Input-Plugin#47]
Hello!

Any plans on supporting OpenXR games?

Flight Simulator 2020 really needs this tool...

Thanks!

**Comment by adamboy7** [comment]
Agreed, just got my hands on a WMR headset. Works great with Steam, but I can't record things like the windows hub or minecraft bedrock sadly. OpenXR would be a welcome addition.

**Comment by baffler** [comment]
It's something I plan on looking into, but there's no guarantee I can make it work. I would like to add the support if I can though

**Comment by ASchneiderBR** [comment]
>It's something I plan on looking into, but there's no guarantee I can make it work. I would like to add the support if I can though

@baffler that would be amazing mate! FS2020 needs it so badly :) the entire FS community will be forever thankful for it.

**Comment by eyildirir** [comment]
@baffler Any updates on this? I'm pretty sure some people that use this addon (including myself) would love to help with this, if we can, since this is a feature a lot of people direly need.

**Comment by J4P032** [comment]
No replies? I am also interested in this one MSFS needs a direct support in OBS as OpenVR has

**Comment by wotupfoo** [comment]
You could try this repo instead. It implements an OBS plugin as an OpenXR Layer capturing the image before lens warping etc. 

https://github.com/Jabbah/OpenXR-Layer-OBSMirror
